### PR TITLE
Make superclass for params

### PR DESCRIPTION
**Pull request by Steven-Roberts**
Starting with L63 to if this is useful

### CODE REVIEW

File: toolbox/+otp/+cusp/CUSPParameters.m [hunk 1]
```diff
@@ -1,4 +1,4 @@
-classdef CUSPParameters
+classdef CUSPParameters < otp.Parameters
     % Parameters for the CUSP problem.
     properties
         % The stiffness parameter $\varepsilon$ for the "cusp catastrophe" model.
```
Comment (AndreyAPopov) [inline toolbox/+otp/+cusp/CUSPParameters.m:4]: greek letters here please

File: toolbox/+otp/+cusp/+presets/Canonical.m [hunk 2]
```diff
@@ -30,22 +30,19 @@
...
-            opts = p.Results;
+            n = p.Results.N;
             
-            params = otp.cusp.CUSPParameters;
-            params.Epsilon = opts.epsilon;
-            params.Sigma = opts.sigma;
-            
-            ang = 2 * pi / opts.N * (1:opts.N).';
-            y0 = zeros(opts.N, 1);
+            ang = 2 * pi / n * (1:n).';
```
Comment (AndreyAPopov) [inline toolbox/+otp/+cusp/+presets/Canonical.m:36]: make unambiguois

File: toolbox/+otp/+protherorobinson/+presets/Canonical.m [hunk 3]
```diff
@@ -8,37 +8,27 @@
...
-            % lambda : numeric(1, 1), default=-200
-            %    The stiffness parameter and eigenvalue of the Jacobian $\lambda$.
-            % phi : function_handle, default=@(t)10-(10+t).*exp(-t)
-            %    The exact solution.
-            % dphi : function_handle, default=@(t)(9+t).*exp(-t)
-            %    The time derivative of $\phi(t)$.
+            % varargin
+            %    A variable number of name-value pairs. The accepted names are
+            %
+            %    - ``Lambda`` – The stiffness parameter and eigenvalue of the Jacobian $\lambda$.
```
Comment (AndreyAPopov) [inline toolbox/+otp/+protherorobinson/+presets/Canonical.m:19]: greek letters

File: toolbox/+otp/+protherorobinson/ProtheroRobinsonParameters.m [hunk 4]
```diff
@@ -1,4 +1,4 @@
-classdef ProtheroRobinsonParameters
+classdef ProtheroRobinsonParameters < otp.Parameters
     % Parameters for the Prothero–Robinson problem.
     properties
         % The stiffness parameter and eigenvalue of the Jacobian $\lambda$.
```
Comment (AndreyAPopov) [inline toolbox/+otp/+protherorobinson/ProtheroRobinsonParameters.m:4]: greek letters

**Review by elswit — APPROVED** [review]

**Review by AndreyAPopov — APPROVED** [review]

**Pull request merged**